tests/test_internals/test_movie_interface.py

```diff
@@ -83,13 +83,15 @@ def assert_nwbfile_conversion(
     converter.run_conversion(
         metadata=metadata, nwbfile_path=nwbfile_path, overwrite=True, conversion_options=dict(Movie=conversion_opts)
     )
-    module_name = module_name if module_name is not None else "acquisition"
     with NWBHDF5IO(path=nwbfile_path, mode="r") as io:
         nwbfile = io.read()
-        mod = nwbfile.modules[module_name]
-        assert module_name in mod
-        if module_description:
-            assert module_description == mod.description
+        if module_name is None:
+            mod = nwbfile.acquisition
+        else:
+            assert module_name in nwbfile.processing
+            mod = nwbfile.processing[module_name]
```

Comment:
maybe it would be better to use `get_module` here

Reply:
get module [here](https://github.com/catalystneuro/nwb-conversion-tools/blob/cfc01eb11868c7d666ec5dca4e6d0ef251bac538/nwb_conversion_tools/utils/conversion_tools.py#L13) creates the processing module if it does not exist. This test expects that module name to exist already.